moto/route53/models.py

```diff
@@ -575,10 +575,9 @@ def list_hosted_zones_by_vpc(self, vpc_id):
                 this_zone = self.get_hosted_zone(zone.id)
                 for vpc in this_zone.vpcs:
                     if vpc["vpc_id"] == vpc_id:
-                        this_id = f"/hostedzone/{zone.id}"
                         zone_list.append(
                             {
-                                "HostedZoneId": this_id,
+                                "HostedZoneId": zone.id,
```

Comment:
Did you verify this against AWS @macnev2013? [The docs](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/route53.html#Route53.Client.get_hosted_zone) for `get_hosted_zone` specifically say that the output should follow this format, but nothing about the format in `list_hosted_zones_by_vpc`

Reply:
Yes. you can check this doc [here](https://docs.aws.amazon.com/Route53/latest/APIReference/API_ListHostedZonesByVPC.html).
Also verified it with terraform.

Reply:
Gotcha. It looks good to me - happy to merge once the unit tests pass

Reply:
Test passed. :)